src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -196,7 +207,7 @@ class FunSpec private constructor(
   }
 
   private fun isExpressionBody(): Boolean {
```

Comment:
What's this used for? I'm tempted to make the callers just check for nullability of asExpressionBody similar to what is happening in the codeWriter interaction above.

Reply:
Have removed this method. It was used to check emitting of return types.